package.json

```diff
@@ -41,7 +41,7 @@
     "pretest": "npm run build",
     "test": "gulp test",
     "test:fast": "npm run build:fast && gulp test",
-    "ci": "npm run test && (catch-uncommitted || echo 'new Date().toISOString().startsWith(\"2019-04-13\") || process.exit(1)' | node)",
```

Comment:
This reverts what I had to do in the previous (unrelated) CLI PR in order to get the Travis build to pass. Possibly what was going wrong was that the version of [prettier](https://www.npmjs.com/package/prettier) was different between my laptop and the Travis build (we don't currently commit `package-lock.json` for the CLI).